public/assets/styles/index.scss

```diff
@@ -0,0 +1,52 @@
+@import url(https://fonts.googleapis.com/css?family=Zilla+Slab:300,300i,400,500,600,700%7COpen+Sans:300,400,600,700,800);
+@import url('https://fonts.googleapis.com/css?family=Open Sans');
```

Comment:
This line of code should not be there again because the first line includes both `Zilla slab` and `Open Sans`. Please remove it.

Reply:
Ok. Thanks for the feedback. Let me implement that.

Reply:
@sulenchy done.